variants/DISCO_F413ZH/variant.h

```diff
@@ -0,0 +1,164 @@
+ *        Pins
+ *----------------------------------------------------------------------------*/
+
+#define PF6  0  // D0
+#define PF7  1  // D1
+#define PG13 2  // D2
+#define PF10 3  // D3
+#define PB6  4  // D4
+#define PE6  5  // D5
+#define PB0  6  // D6
+#define PC13 7  // D7
+#define PA4  8  // D8
+#define PB8  9  // D9
+#define PA15 10 // D10 - SPI_SSEL
+#define PB5  11 // D11 - SPI_MOSI, ISM43362_WIFI_MOSI
+#define PB4  12 // D12 - SPI_MISO, ISM43362_WIFI_MISO 
+#define PB12 13 // D13 - SPI_SCK, LED_GREEN, ISM43362_WIFI_SCLK
+#define PB11 14 // D14 - I2C_SDA
+#define PB10 15 // D15 - I2C_SCL
+#define PC0  16 // D16/A0
+#define PA1  17 // D17/A1
+#define PA2  18 // D18/A2
+#define PA5  19 // D19/A3
+#define PB1  20 // D20/A4
+#define PC4  21 // D21/A5
+#define PG14 22 // D22 - STLink Tx
+#define PG9  23 // D23 - STLink Rx
+// not on connector
+#define PE3  24 // D24 - LD4, LED1_RED
+#define PC5  25 // D25 - LD3, LED2_GREEN
+#define PA0  26 // D26 - User button
+#define PG11 27 // D27 - ISM43362_WIFI_NSS
+#define PH1  28 // D28 - ISM43362_WIFI_RESET
+#define PG12 29 // D29 - ISM43362_WIFI_DATAREADY
+#define PB15 30 // D30 - ISM43362_WIFI_WAKEUP
+
+
+// This must be a literal
+// It is used with preprocessor tests (e.g. #if NUM_DIGITAL_PINS > 3)
+// so an enum will not work.
+#define NUM_DIGITAL_PINS        31
+
+// Allow to define Arduino style alias for analog input pin number --> Ax
+#define NUM_ANALOG_INPUTS       6
+#define NUM_ANALOG_FIRST        16
+
+// On-board LED pin number
+#define LED_BUILTIN             PB12
+#define LED_GREEN               LED_BUILTIN
+#define LED_GREEN2              PC5
+#define LED_RED                 PE3
+
+// On-board user button
+#define USER_BTN                PA0
+
+// Below SPI and I2C definitions already done in the core
+// Could be redefined here if differs from the default one
+// SPI Definitions
+#define PIN_SPI_SS              10 // Default for Arduino connector compatibility
+#define PIN_SPI_MOSI            11 // Default for Arduino connector compatibility
+#define PIN_SPI_MISO            12 // Default for Arduino connector compatibility
+#define PIN_SPI_SCK             13 // Default for Arduino connector compatibility
+
+// I2C Definitions
+#define PIN_WIRE_SDA            14 // Default for Arduino connector compatibility
+#define PIN_WIRE_SCL            15 // Default for Arduino connector compatibility
+
+// I2C timing definitions (optional), avoid time spent to compute if defined
+// * I2C_TIMING_SM for Standard Mode (100kHz)
+// * I2C_TIMING_FM for Fast Mode (400kHz)
+// * I2C_TIMING_FMP for Fast Mode Plus (1000kHz)
+//#define I2C_TIMING_SM           0x00000000
+//#define I2C_TIMING_FM           0x00000000
+//#define I2C_TIMING_FMP          0x00000000
+
+// Timer Definitions (optional)
+// Use TIM6/TIM7 when possible as servo and tone don't need GPIO output pin
+#define TIMER_TONE              TIM8
+#define TIMER_SERVO             TIM9
```

Comment:
TIM6 and TIM7 are generally selected as they are very basic and don't have output pins, which is enough for Tone and Servo. As they have no output there is no entry in` PinMap_PWM[]`.
As they exist on F423ZH I propose to keep them.

Reply:
Understood, will do so.